docs/guides/use-connectors-in-hybrid-mode.md

```diff
@@ -79,6 +81,27 @@ The `X` is normalized to the environment variable Connector name. For example, t
 `HTTP REST` name becomes `HTTP_REST`, or the [Kafka Consumer Connector](https://github.com/camunda/connectors/blob/main/connectors/kafka/src/main/java/io/camunda/connector/kafka/inbound/KafkaExecutable.java#L20) name
 becomes `KAFKA_CONSUMER`. Therefore, to override it one would need to pass in the `CONNECTOR_KAFKA_CONSUMER_TYPE=xxx` environment variable.
 
+## Using SaaS secrets
+
+If you add the `Secrets` scope to your API client, you can access cluster [secrets](../components/connectors/use-connectors/index.md#using-secrets) in a hybrid setup.
+
+Enable the SaaS secret provider via an environment variable or in your application config file:
+
+**Environment variable:**
+
+```
+CAMUNDA_CONNECTOR_SECRETPROVIDER_CONSOLE_ENABLED = true
+```
+
+**Properties file:**
```

Comment:
Do we mean "properties" file and "application config" file interchangeably here?